Source/WebCore/rendering/RenderTextControlSingleLine.cpp

```diff
@@ -366,15 +366,21 @@ void RenderTextControlSingleLine::autoscroll(const IntPoint& position)
 
 int RenderTextControlSingleLine::scrollWidth() const
 {
-    if (auto innerTextElement = this->innerTextElement(); innerTextElement && innerTextElement->renderer())
-        return innerTextElement->renderer()->scrollWidth();
+    if (auto inner = this->innerTextElement() ? innerTextElement()->renderer() : nullptr) {
```

Comment:
This is slightly different from the blink patch as you are getting scrollWidth off of the renderer instead of the element, but I guess it's fine.
I would write it like this (there was also a typo in blink's commit.

if (auto* innerTextRenderer = innerTextElement() ? innerTextElement()->renderer() : nullptr) {
   // Adjust scrollWidth to include input element horizontal paddings and decoration width.
   auto adjustment = clientWidth() - innerTextRenderer->clientWidth();
   return innerTextRenderer->scrollWidth() + adjustment;
 }
(same applies to scrollHeight)

Reply:
Heading back home. Will update once I have access to personal laptop. 👍

Reply:
Done.